artifactory/utils/docker/buildinfo.go

```diff
@@ -316,7 +317,44 @@ func getConfigLayer(imageId string, searchResults map[string]utils.ResultItem, s
 }
 
 // Search for image layers in Artifactory.
-func searchImageLayers(imageId, imagePathPattern string, serviceManager *artifactory.ArtifactoryServicesManager) (map[string]utils.ResultItem, error) {
+func searchImageLayers(builder *buildInfoBuilder, imagePathPattern string, serviceManager *artifactory.ArtifactoryServicesManager) (map[string]utils.ResultItem, error) {
```

Comment:
I suggest adding a unit test, which will checks the parsing of the fat manifest.

Reply:
We don't parse but marshal the expected fat-manifest format,
TestDockerFatManifestPull should be all we need to check this